src/server/terrain/SurfaceMap.zig

```diff
@@ -64,6 +66,10 @@ pub const MapFragment = struct { // MARK: MapFragment
 	pub const mapSize = 1 << mapShift;
 	pub const mapMask = mapSize - 1;
 
+	const biomeDataSize = mapSize*mapSize*@sizeOf(u32);
+	const heightDataSize = mapSize*mapSize*@sizeOf(i32);
+	const originalHeightDataSize = mapSize*mapSize*@sizeOf(i32);
```

Comment:
In my opinion this is an implementation detail of the particular compression method and should remain locally in the compression/decompression functions like it was before.